src/js/components/DateInput/doc.js

```diff
@@ -59,6 +59,9 @@ export const doc = DateInput => {
       PropTypes.string,
       PropTypes.arrayOf(PropTypes.string),
     ]).description('The date or date range value(s) in ISO8601 format.'),
+    disabled: PropTypes.bool.description(
+      'Whether the field should look disabled.',
```

Comment:
```suggestion
      'Whether the Datenput is disabled.',
```

Reply:
sorry didnt see before can we maybe change the description so it matches the other components..